Controller/ExceptionController.php

```diff
@@ -195,7 +195,10 @@ protected function getFormat(Request $request, $format)
     {
         try {
             $formatNegotiator = $this->container->get('fos_rest.exception_format_negotiator');
-            $format = $formatNegotiator->getBestFormat($request) ?: $format;
+            $accept = $formatNegotiator->getBest('', array());
```

Comment:
This if checks $accept value, so if I move this line which defines $accept, how could the condition be true?...